token/js/src/extensions/cpiGuard/instructions.ts

```diff
@@ -70,7 +70,11 @@ function createCpiGuardInstruction(
     const keys = [{ pubkey: account, isSigner: false, isWritable: true }];
     keys.push({ pubkey: authority, isSigner: !multiSigners.length, isWritable: false });
     for (const signer of multiSigners) {
-        keys.push({ pubkey: signer.publicKey, isSigner: true, isWritable: false });
+        keys.push({
+            pubkey: signer instanceof PublicKey ? signer : signer.publicKey,
+            isSigner: true,
+            isWritable: false,
+        });
     }
```

Comment:
This predates your PR, but could all of the instances that *don't* call `addSigners` call that instead? so this would become:

```
const keys = addSigners(
        [
            { pubkey: account, isSigner: false, isWritable: true },
        ],
        authority,
        multiSigners
);
```

Reply:
Updated!